rotkehlchen/chain/evm/decoding/safe/decoder.py

```diff
@@ -97,6 +97,33 @@ def _decode_execution_success(self, context: DecoderContext) -> DecodingOutput:
         )
         return DecodingOutput(event=event)
 
+    def decode_safe_creation(self, context: DecoderContext) -> DecodingOutput:
+        num_owners = (len(context.tx_log.data) // 32) - 5  # data has 5 elements that are not the addresses  # noqa: E501
+        owners = []
+        # chunk the data and iterate in reverse order. First elements are the owners and then
+        # comes the threshold
+        for index, entry in enumerate(reversed([context.tx_log.data[i:i + 32] for i in range(0, len(context.tx_log.data), 32)])):  # noqa: E501
+            if index < num_owners:
+                owners.append(hex_or_bytes_to_address(entry))
+            elif index == num_owners:
+                threshold = hex_or_bytes_to_int(entry)
+            else:
+                break
+
+        event = self.base.make_event_next_index(
+            tx_hash=context.transaction.tx_hash,
+            timestamp=context.transaction.timestamp,
+            event_type=HistoryEventType.INFORMATIONAL,
+            event_subtype=HistoryEventSubType.NONE,
+            asset=A_ETH,
+            balance=Balance(),
+            location_label=context.transaction.from_address,
+            notes=f'Created a new safe with a threshold of {threshold} and owners {" , ".join(owners)}',  # noqa: E501
```

Comment:
Notice there is 2 changes here

```suggestion
            notes=f'Create a new safe with a threshold of {threshold} and owners {",".join(owners)}',  # noqa: E501
```